oligo_designer_toolsuite/database/_oligo_database_attributes.py

```diff
@@ -3,216 +3,125 @@
 ############################################
 
 import warnings
-from typing import List, Union
+from typing import List, Union, Tuple
 
 from Bio.SeqUtils import MeltingTemp as mt
 from Bio.SeqUtils import Seq, gc_fraction
 from seqfold import dg
 
 from oligo_designer_toolsuite._constants import _TYPES_SEQ
 from oligo_designer_toolsuite.database import OligoDatabase
-from oligo_designer_toolsuite.utils import check_if_key_exists, check_if_list
+from oligo_designer_toolsuite.utils import check_if_list, flatten_attribute_list
 
 ############################################
 # Attrubite Calculation Class
 ############################################
 
 
 class OligoAttributes:
```

Comment:
All the computations of the attribues could be parallelized region-wise to speed up the computations.

Reply:
True, we will address it in the oligo attribute issue